lib/matplotlib/tests/test_axes.py

```diff
@@ -5935,6 +5935,33 @@ def test_tick_padding_tightbbox():
     assert bb.y0 < bb2.y0
 
 
+def test_inset():
+    """Regression test: failure described in for issue #14275"""
```

Comment:
Please don't reference issues in the comments..  Every commit adds a notification message in the issue ;-)

Reply:
Yes, I noticed that! Sorry. I think it's valuable to reference the issue in regression tests, but next time I won't include the `#`, to stop github picking it up.

Reply:
#12475 mentions a doc problem, so the test can't really be testing for that.  I'd rather just explain what you're testing in the docstring.
Also "Regression test" basically adds no information: most tests are regression tests...